charter.html

```diff
@@ -704,16 +690,39 @@ <h2 id=about>About this Charter</h2>
 
 <script>
 document.addEventListener("DOMContentLoaded", function() {
+  const headingElements = ["h1", "h2", "h3", "h4", "h5", "h6"];
+  let sectionCounters = [NaN, 0, 0, 0, 0, 0, 0];
   document.querySelectorAll("[id]").forEach(function(heading) {
-    let permalink = document.createElement("a");
+    const permalink = document.createElement("a");
     permalink.href = "#" + heading.getAttribute("id");
     permalink.className = "permalink";
-    permalink.appendChild(document.createTextNode("#"));
+    let sigil = "";
+    if (headingElements.includes(heading.localName)) {
+      const headingLevel = parseInt(heading.localName.substring(1), 10);
+      console.log("saw an " + heading.localName + "headingLevel " + headingLevel);
```

Comment:
Was this temporary logging you meant to remove?

Reply:
Yes.